staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1119,6 +1119,10 @@ func (c *Cacher) dispatchEvent(event *watchCacheEvent) {
 	// Since add() can block, we explicitly add when cacher is unlocked.
 	// Dispatching event in nonblocking way first, which make faster watchers
 	// not be blocked by slower ones.
+	//
+	// Note: if we ever decide to cache the serialization of bookmark events,
+	// we will also need to modify the watchList encoder
+	// introduced in https://github.com/kubernetes/kubernetes/pull/127441
```

Comment:
Let's not add the PR number - let's just use the exact name we use for it: `watchListEncoder`

Please also be more prescriptive of what/how would need to be updated.
(we will need determinism of object -> serialization that doesn't depend on any context of the request itself)